django/contrib/admin/options.py

```diff
@@ -670,6 +672,18 @@ class ModelAdmin(BaseModelAdmin):
     actions_selection_counter = True
     checks_class = ModelAdminChecks
 
+    def __init_subclass__(cls, **kwargs) -> None:
+        super().__init_subclass__(**kwargs)
+        if cls.__dict__.get("list_select_related") is True:
+            # RemovedInDjango70Warning: when the deprecation ends, raise a
+            # ValueError.
+            warnings.warn(
+                "Setting ModelAdmin.list_select_related to True is deprecated. "
+                "Use False or a list or tuple of fields to fetch instead.",
```

Comment:
I think we should update https://github.com/django/django/blob/906a51e125c3007f86d42b81072a1dad7149af05/docs/ref/checks.txt#L779-L780.